conda_forge_tick/migrators/arch.py

```diff
@@ -180,17 +174,18 @@ def __init__(
     def filter(self, attrs: "AttrsTypedDict", not_bad_str_start: str = "") -> bool:
         if super().filter(attrs):
             return True
-        muid = frozen_to_json_friendly(self.migrator_uid(attrs))
         for arch in self.arches:
             configured_arch = (
                 attrs.get("conda-forge.yml", {}).get("provider", {}).get(arch)
+            ) or (
+                attrs.get("conda-forge.yml", {}).get("build_platform", {}).get(arch)
+                not in [None, arch]
             )
-            if configured_arch:
-                return muid in _sanitized_muids(
-                    attrs.get("pr_info", {}).get("PRed", []),
-                )
```

Comment:
I am confused as to why we no longer need this block of code any more. The call to `super().filter` above does not check if the PR has already been made. This change will force the bot to reissue arch PRs to feedstocks if they merged an arch PR but then subsequently removed arch builds for w/e reason.